src/organization/screens/organization-profile.screen.js

```diff
@@ -109,83 +87,82 @@ class OrganizationProfile extends Component {
 
   handleActionSheetPress = index => {
     if (index === 0) {
-      openURLInView(this.props.organization.html_url);
+      openURLInView(this.props.org._entityUrl);
     }
   };
 
   render() {
     const {
-      organization,
+      orgId,
+      org,
       members,
-      isPendingOrg,
-      isPendingMembers,
+      membersPagination,
       navigation,
       language,
     } = this.props;
     const { refreshing } = this.state;
     const initialOrganization = this.props.navigation.state.params.organization;
     const organizationActions = [translate('common.openInBrowser', language)];
 
+    if (!org) {
+      return (
+        <Text>
+          Loading organization {orgId} .. TODO: Make me look nicer
+        </Text>
+      );
+    }
```

Comment:
We should instead be drawing the screen with loaders everywhere where we don't have the info in store yet